tests/Unit/CssInlinerTest.php

```diff
@@ -430,11 +440,11 @@ public function wbrTagDataProvider()
      *
      * @dataProvider wbrTagDataProvider
      */
-    public function emogrifyByDefaultRemovesWbrTag($html)
+    public function renderByDefaultRemovesWbrTag($html)
     {
```

Comment:
Should the method name be `inlineCss...` rather than `render...`, since it's the `inlineCss` method that removes these elements?

Reply:
Done.